androidApp/src/main/java/com/mbta/tid/mbta_app/android/component/StopListRow.kt

```diff
@@ -62,26 +74,26 @@ fun StopListRow(
     alertSummaries: Map<String, AlertSummary?> = emptyMap(),
     connectingRoutes: List<Route>? = null,
     disruption: UpcomingFormat.Disruption? = null,
-    firstStop: Boolean = false,
     isTruncating: Boolean = false,
-    lastStop: Boolean = false,
+    stopRowStyle: StopRowStyle = StopRowStyle.MidLineStop,
     onOpenAlertDetails: (Alert) -> Unit = {},
     showDownstreamAlert: Boolean = false,
     showStationAccessibility: Boolean = false,
     targeted: Boolean = false,
     trackNumber: String? = null,
+    descriptor: @Composable () -> Unit = {},
     rightSideContent: @Composable RowScope.(Modifier) -> Unit = {},
 ) {
     val context = LocalContext.current
 
     val stateBefore =
-        when {
-            firstStop -> RouteLineState.Empty
+        when (stopRowStyle) {
+            StopRowStyle.FirstLineStop -> RouteLineState.Empty
```

Comment:
I would expect `StandaloneStop` to give `stateBefore` `Empty` and `stateAfter` `Empty`, although the new `RouteLine` conditional probably can‘t be pushed down into the `RouteLine` component itself so it may not actually make a difference.

Reply:
I could almost push it down into the RouteLine, except the afterState is separately used when there is a disruption and we don't use the RouteLine component. Opted to add a case for standalone stops here via the new `includeLineDiagram` prop.